viz/tap/api/handlers.go

```diff
@@ -171,6 +171,7 @@ func (h *handler) handleTap(w http.ResponseWriter, req *http.Request, p httprout
 	}
 
 	serverStream := serverStream{w: flushableWriter, req: req, log: h.log}
+	//nolint:staticcheck
```

Comment:
The broader context is that this method is served by our weird proto-over-http protocol and not by a grpc server.  Since there's no grpc server which serves this method, it was argued that this should be marked as deprecated.  I still don't agree with that logic and I don't think it should have been marked as deprecated in the first place; but here we are.